tests/virt_autotest/kubevirt_tests_server.pm

```diff
@@ -220,6 +220,15 @@ sub install_kubevirt_packages {
         foreach (split(' ', $virt_manifests_pkgs), $virt_tests_pkg) {
             $pkgs_from_incident_repo += 1 if (script_output("zypper info $_ | awk -F': ' '/^Repository/{print \$2}'") =~ /^TEST_/);
         }
+        # Patch the kubevirt-operator manifest to use images from the SUSE maintenance registry
+        my $incident_id = get_required_var('INCIDENT_ID');
+        my $manifest = "/usr/share/kube-virt/manifests/release/kubevirt-operator.yaml";
+        my $src_repo = "registry.suse.com/suse/";
+        my $dst_repo = "registry.suse.de/suse/maintenance/$incident_id/containerfile/suse/";
+        my $replace_registry = "sed -i 's|$src_repo|$dst_repo|g' $manifest";
+
+        assert_script_run($replace_registry);
```

Comment:
If this command will run once only, I would not recommend creating a new variable.